airgun/entities/sync_templates.py

```diff
@@ -0,0 +1,53 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep
+from airgun.navigation import navigator
+from airgun.views.template import SyncTemplatesView
+
+
+class SyncTemplatesEntity(BaseEntity):
+    endpoint_path = '/template_syncs'
+
+    def import_templates(self, values):
+        """Import Templates entity"""
+        view = self.navigate_to(self, 'import')
+        view.fill(values)
+        view.submit.click()
+
+    def export_templates(self, values):
+        """Export Templates entity"""
+        view = self.navigate_to(self, 'export')
+        view.fill(values)
+        view.submit.click()
+
+
+@navigator.register(SyncTemplatesEntity, 'All')
+class ShowMainPageActionNavigation(NavigateStep):
+    """Navigate to the page thats main Import Export Templates page"""
+    VIEW = SyncTemplatesView
+
+    def step(self, *args, **kwargs):
+        self.view.menu.select('Hosts', 'Sync Templates')
+
+
+@navigator.register(SyncTemplatesEntity, 'import')
```

Comment:
```suggestion
@navigator.register(SyncTemplatesEntity, 'Import')
```
No idea why we have to do it, but rest of navigators in Airgun following the sae approach.
It would be good if we keep it that way :P

Reply:
Strictly speaking, we don't. Navigation endpoints could be anything, `fdh874rfyj` is as good as any. Capitalization is just a convention that we follow.

But yes, being consistent and following established conventions is definitely desired.